### ISSUE TITLE
use_websocket errors on route change

### ISSUE THREAD

**Issue by BakerNet** [Synphonyte/leptos-use#22]
When using `use_websocket` inside of a route, there are console errors on route change away.

Example console trace (hashes/filenames omitted):
```
Uncaught RuntimeError: unreachable
    at __rust_start_panic
    at rust_panic
    at std::panicking::rust_panic_with_hook
    at std::panicking::begin_panic_handler::{{closure}}
    at std::sys_common::backtrace::__rust_end_short_backtrace
    at rust_begin_unwind
    at core::panicking::panic_fmt
    at core::option::expect_failed
    at core::option::Option<T>::expect
    at leptos_reactive::stored_value::StoredValue<T>::get_value
```

The trace would suggest a lifetime issue with one of the stored values in `use_websocket`.  Unfortunately the trace doesn't help identify which stored value it might be.

Example repo to demonstrate the error:  https://github.com/BakerNet/leptos-use-websocket-error

**Comment by maccesch** [comment]
Thanks for the excellent error report! Fixed in cac1b2c2d830444dad62546997883362aaefa71f.

Also released as 0.6.3.

**Comment by BakerNet** [comment]
> Thanks for the excellent error report! Fixed in [cac1b2c](https://github.com/Synphonyte/leptos-use/commit/cac1b2c2d830444dad62546997883362aaefa71f).
> 
> Also released as 0.6.3.

That was fast!  Verified fixed on my end after update.
